### PR TITLE
feat(line-ripple): Convert JS to TypeScript

### PR DESCRIPTION
**Pull request by acdvorak**
Refs #4225

### CODE REVIEW

File: packages/mdc-line-ripple/adapter.ts [hunk 1]
```diff
@@ -21,57 +21,40 @@
...
-  setStyle(propertyName, value) {}
+  setStyle(propertyName: string, value: string): void;
 
   /**
    * Registers an event listener on the line ripple element for a given event.
-   * @param {string} evtType
-   * @param {function(!Event): undefined} handler
    */
-  registerEventHandler(evtType, handler) {}
+  registerEventHandler(evtType: string, handler: EventListener): void;
```
Comment (moog16) [inline packages/mdc-line-ripple/adapter.ts]: this and deregister should be using `EventType` and `SpecificEventListener`

File: packages/mdc-line-ripple/index.ts [hunk 2]
```diff
@@ -54,27 +44,27 @@ class MDCLineRipple extends MDCComponent {
...
 
   /**
-   * Sets the transform origin given a user's click location. The `rippleCenter` is the
-   * x-coordinate of the middle of the ripple.
-   * @param {number} xCoordinate
+   * Sets the transform origin given a user's click location.
+   * The `rippleCenter` is the x-coordinate of the middle of the ripple.
    */
-  setRippleCenter(xCoordinate) {
+  setRippleCenter(xCoordinate: number): void {
```
Comment (kfranqueiro) [inline packages/mdc-line-ripple/index.ts]: Is this `: void` even needed?

File: packages/mdc-line-ripple/adapter.ts (hunk 1 above)
Comment (acdvorak) [reply packages/mdc-line-ripple/adapter.ts]: Done

File: packages/mdc-line-ripple/index.ts (hunk 2 above)
Comment (acdvorak) [reply packages/mdc-line-ripple/index.ts]: Removed

**Pull request merged**